agreements/agreements-service.yaml

```diff
@@ -412,6 +416,19 @@ components:
           items:
             $ref: '#/components/schemas/LotSummary'
 
+    ContactDetail:
+      type: object
+      description: Details of a contact. Initially email only, will be extended in future as additional contact channels are added (e.g. phone number)
+      properties:
+        email:
+          type: string
+          format: email
+          description: Email address
+          example: "Category.Manager@crowncommercial.gov.uk"
+        type:
+          type: string
+          description: The 'type' of the contact in business terms, used to identify the correct contact for a particular purpose.
+          example: "CCSCategoryManager"
```

Comment:
Assume this could not be an enum at this stage, i.e. we don't know all the values?

Reply:
Exactly. I wanted to keep it flexible